src/reaper_osara.cpp

```diff
@@ -2758,6 +2758,47 @@ void cmdRemoveTracks(Command* command) {
 	cmdhRemoveTracks(command->gaccel.accel.cmd);
 }
 
+void cmdRemoveOrCopyAreaOfItems(Command* command) {
+	double start, end;
+	GetSet_LoopTimeRange(false, true, &start, &end, false);
+	int selItems = CountSelectedMediaItems(nullptr);
+	if(start == end) {
+		outputMessage(translate("No time selection"));
```

Comment:
Are we certain all of these actions really do nothing when there's no time selection? Do any or all of them just use the item selection if there's no time selection?

Reply:
I can't get them to do anything if there is no time selection.
With some further testing however, I did find that when no items are selected, "Cut/Remove selected area of items" acts as if all items are selected.  Copy selected area of items however does nothing when no items are selected.

Thanks for the consistent and intuitive behaviour Cockos! :)